api/swagger.yaml

```diff
@@ -8044,6 +8068,7 @@ paths:
             items:
               description: "Describes a permission the user has to accept upon installing the plugin."
               type: "object"
+              title: "PluginPrivilege"
```

Comment:
`PluginPrivilegeResponse`

Reply:
as response is an Array here, maybe `PluginPrivilegeItem` ?

Reply:
sounds good, I missed that it was an item